packages/p2p/src/components/orders/order-table/order-table.jsx

```diff
@@ -27,11 +27,11 @@ const OrderTable = observer(({ showDetails }) => {
     const is_active_tab = general_store.order_table_type === order_list.ACTIVE;
     return (
         <React.Fragment>
-            <div className='orders__header'>
+            <div className='orders-tab__header'>
```

Comment:
Is it a tab or table?

Reply:
It's related to the orders tab,  we have the same "orders_header" class on Buy/Sell tab, and because of that we have some issues with design so I  have to change the name of this class